.travis.yml

```diff
@@ -1,25 +1,48 @@
-dist: trusty
```

Comment:
This diff is very confusing.  Could you please try to keep it more in the same order as it was originally?

Reply:
Will keep original order.